server/src/middleware/user-is-authorized-by-param.middleware.ts

```diff
@@ -2,7 +2,7 @@ import {Request, Response} from 'express';
 import {wrapResponse} from '../functions/response-wrapper';
 import {verifyToken} from '../functions/verify-token.func';
 
-export async function userIsAuthorizedByParam(req: Request, res: Response, next: any): Promise<void> {
+export async function userIsAuthorizedByParam(req: Request, res: Response, next: unknown): Promise<void> {
```

Comment:
```suggestion
export async function userIsAuthorizedByParam(req: Request, res: Response, next: NextFunction): Promise<void> {
```

Bitte überprüfen - sollte aber das sein (import fehlt noch)

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express/index.d.ts#L107